gridt/src/app/home/expandable/expandable.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, Input } from "@angular/core";
```

Comment:
The file and folder name should be renamed still.

Reply:
this has given me some issues yesterday, but i will try again today.

Reply:
Turns out, i forgot to redo ionic serve...